solutions/fully-configurable/catalogValidationValues.json.template

```diff
@@ -0,0 +1,8 @@
+{
+  "ibmcloud_api_key": $VALIDATION_APIKEY,
+  "existing_resource_group_name": $PREFIX,
```

Comment:
This now needs to be an existing group in our account. Suggest to use the one we use in pr_test.go: `geretain-test-event-notifications`